ani-cli

```diff
@@ -289,9 +289,14 @@ anime_selection () {
 		menu_line_strong "exit" "q"
 		prompt "Enter choice"
 		choice="$REPLY $REPLY2"
+        while ! [ "$choice" -eq "$choice" ] 2>/dev/null || [ "$choice" -lt 1 ] || [ "$choice" -ge "$count" ] || [ "$choice" = " " ] ;
+        do
+            [ "$choice" = "q " ] && die
```

Comment:
I thick choice is not an error it shouldn't die instead `exit 0` is more appropriate

Reply:
Nice find! I also think that exiting with 0 is what the script should do.